litestar/cli/main.py

```diff
@@ -16,13 +16,14 @@
 
 
 @click.group(cls=LitestarExtensionGroup, context_settings={"help_option_names": ["-h", "--help"]})
-@click.option("--app", "app_path", help="Module path to a Litestar application")
+@click.option("--app", "app_path", help="Module path to a Litestar application", is_eager=True)
```

Comment:
This is a nice solution! Didn't even know this flag existed :)

Reply:
TIL too